sqlx-core/Cargo.toml

```diff
@@ -43,6 +44,8 @@ futures-util = { version = "0.3.4", default-features = false }
 generic-array = { version = "0.12.3", default-features = false, optional = true }
 hex = "0.4.2"
 hmac = { version = "0.7.1", default-features = false, optional = true }
+ipnetwork = { version = "0.16.0", default-feature = false, optional = true }
+libc = { version = "0.2.68", default-feature = false, optional = true }
```

Comment:
If we need `libc` I think I would just let it be non-optional so we can use `ipnetwork` as the feature name